code/Backend/flask-backend/app.py

```diff
@@ -0,0 +1,62 @@
+from flask import Flask, request, jsonify
```

Comment:
Do we need a README.md file for backend as well? Just a question

Reply:
I think 1 read.me should suffice that has instructions on how to run both the backend and front end code on a local machine.

Reply:
When I created the react project it automatically came with a READme - we can always add another one or move it